jib-core/src/main/java/com/google/cloud/tools/jib/http/Authorizations.java

```diff
@@ -35,5 +37,10 @@ public static Authorization withBasicCredentials(String username, String secret)
     return new Authorization("Basic", token);
   }
 
+  /** Creates an {@link Authorization} with a base64-encoded {@code username:password} string. */
```

Comment:
do we/can we ensure this is only used for docker-config consumed credentials?

Reply:
This is intended to be used with any basic authorization token, which are in the form of `username:password`.

Reply:
oh yeah, I didn't really remember if Authorization was part of the api, but I mean are we only creating Authorizations withBasicToken when using docker-config?

Reply:
Yea, the only place that uses this now is in the `DockerConfigCredentialRetriever`.